src/dropdown.ts

```diff
@@ -1250,16 +1250,14 @@ export default class Dropdown extends LitElement implements FormControl {
 
   #isOverflowTest = false;
 
-  // Set in the `value` setter and used in `#onOptionsSelectedChange()` to prevent
-  // the latter from adding duplicate values to `value`.
-  //
-  // Also set in `#onDefaultSlotChange()` to prevent `#onOptionsSelectedChange()` from
-  // resetting the user's filter query back to an empty string.
-  #isProgrammaticSelection = false;
```

Comment:
`#isProgrammaticSelection` is now only set in one place. That let me name give it a more descriptive name.